src/javascripts/components/blog/article/Article.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Header from './Header'
+import Text from './Text'
+import Meta from './Meta'
+import Footer from './Footer'
+// import posts from '../../../data/posts'
+// console.log(posts)
+//need to figure out how to link to this page...
+
+class Article extends React.Component {
+  constructor() {
+    super()
+    this.state = { post: {} }
+  }
+// pull post into state 
+  componentWillMount() {
+    const postId = 1
+    this.setState({
+      // post: posts.find(item => item.id == postId) 
+    })
+  }
+// pass into components to render data
+// then do the author
+  render() {
+    return (
+      <main className="article--container">
+        <article>
+          <Header />
+          <Text />
+        </article>
+        <Meta />
+        <Footer />
```

Comment:
I'd put `<Meta />` and `<Footer />` inside the `<article>` -- those elements are still part of the article content